app/controllers/organizations/questions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Organizations::QuestionsController < Organizations::BaseController
+  before_action :verified_staff
+
+  def new
+    @organization = current_user.organization
+    @form = @organization.form || Form.create(organization: @organization, name: "#{@organization.name}'s Application Form")
```

Comment:
```suggestion
    @form = @organization.form || Form.create(name: "#{@organization.name}'s Application Form")
```
This would be enough without specifying the organization since ActsAsTenant should auto set that.